### PR TITLE
targethud

### PR DESCRIPTION
**Pull request by larryngton**
;3

### CODE REVIEW

**Comment by larryngton** [comment]
> bad

go and fuck yourself

**Comment by DernChess** [comment]
> > bad
> 
> go and fuck yourself


image

File: src/routes/hud/elements/targethud/TargetHud.svelte [hunk 1]
```diff
@@ -77,14 +54,17 @@
 <style lang="scss">
     @import "../../../../colors.scss";
 
-    .targethud {
+    .targethud {                        // WARNING SHITCODE AHEAD
```
Comment (liquidsquid1) [inline src/routes/hud/elements/targethud/TargetHud.svelte:57]: Unneeded comment.

File: src/routes/hud/elements/targethud/TargetHud.svelte [hunk 2]
```diff
@@ -1,9 +1,8 @@
 <script lang="ts">
-    import ArmorStatus from "./ArmorStatus.svelte";
     import {listen} from "../../../../integration/ws.js";
     import type {PlayerData} from "../../../../integration/types";
     import {REST_BASE} from "../../../../integration/host";
-    import {fly} from "svelte/transition";
+    import {fade, fly} from "svelte/transition";
```
Comment (liquidsquid1) [inline src/routes/hud/elements/targethud/TargetHud.svelte:5]: Unused import fly, replace the transitions please.

Comment (DernChess) [reply src/routes/hud/elements/targethud/TargetHud.svelte:5]: > Unused import fly, replace the transitions please.

bro trying to be blaubeere

File: src/routes/hud/elements/targethud/TargetHud.svelte (hunk 1 above)
Comment (larryngton) [reply src/routes/hud/elements/targethud/TargetHud.svelte:57]: the fuck is unneeded?? its a warning

File: src/routes/hud/elements/targethud/TargetHud.svelte (hunk 2 above)
Comment (larryngton) [reply src/routes/hud/elements/targethud/TargetHud.svelte:5]: alright this one is correct though

**Comment by DernChess** [comment]
.

**Pull request merged**